conf/nginx.conf.server.template

```diff
@@ -18,41 +18,12 @@ resolver ${NAMESERVER} ipv6=off valid=10s;
 
 # ====================== Expiration Rules ====================== #
 
-## Configures expiration for upstream content based on content type.
-map $upstream_http_content_type $content_type_expiry {
-    default                                 5s;
-    'application/json'                      5s;
-    'application/pdf'                       max;            # max means 10 years to nginx
-    'application/octet-stream'              max;
-    'binary/octet-stream'                   max;
-    'image/jpeg'                            max;
-    'image/gif'                             max;
-    'image/png'                             max;
-    'image/bmp'                             max;
-    'image/webp'                            max;
-    'video/webm'                            max;
-    'video/ogg'                             max;
-    'video/mp4'                             max;
-    'application/binary'                    -1s;            # used by HLS streams
-    '~*mpegurl'                             -1s;
-}
-
-map $host $host_expiry {
-    default                     $content_type_expiry;
-    # foo.com                   30d;                        # example of expiry by host
-    "~*uplynk.com"              -1s;                        # known HLS stream provider
-}
 
-map $uri $uri_expiry {
-    default                     $host_expiry;
-    # "~*.gifv"                 30d;                        # example of expiry by uri/file extension
+map $request_method $cache_override_expiry {
+    'OPTIONS'                   1s;
```

Comment:
My fix here is to ensure we set the expiry of OPTIONS header to a low value like 1s. This is what allows nginx to cache the OPTIONS request, which can be used when the device is offline. 

I've set this to a low value to avoid impacting any web applications while the device is online. When offline, the OPTIONS request will continue to pull from cache (even after 1s).